src/api.js

```diff
@@ -0,0 +1,70 @@
+import axios from "axios";
+import { formatDate } from "./utils/formatDate";
+
+const ncNewsApi = axios.create({
+    baseURL: 'https://val-northcoders.onrender.com/api',
```

Comment:
Really like the use of a base url here, It makes switching hosts much easier and keeps the code super clean 👍